internal/parser/scanner/scanner.go

```diff
@@ -112,8 +111,10 @@ func (s *scanner) Next() token.Token {
 		return scanUKeyword(s)
 	case 'V':
 		return scanVKeyword(s)
-	// case ' ':
-	// 	return scanSpace(s)
+	case 'W':
+		return scanWKeyword(s)
+	case ' ':
```

Comment:
`' '` is not the only whitespace that is possible

Reply:
Changed.